R/cache.R

```diff
@@ -267,8 +267,8 @@ cache_epidata_call <- function(epidata_call, fetch_args = fetch_args_list()) {
     if (!is.key_missing(cached)) {
       cli::cli_warn(
```

Comment:
Aside: could we make this a `cli_inform` or maybe just silent (or silent + opt-in verbose cache setting?).

Reply:
I made a standalone issue for this, probably a good idea